metals/src/main/scala/scala/meta/internal/pantsbuild/commands/BloopZipkinTraceProperties.scala

```diff
@@ -0,0 +1,18 @@
+package scala.meta.internal.pantsbuild.commands
+
+import scala.meta.internal.pantsbuild.commands.BloopGlobalSettings.Property
+
+object BloopZipkinTraceProperties {
```

Comment:
Can we move all these to `ZipkinUrls`? Note that it lives in the `internal.zipkin` package to make it easier to configure for native-image